Exception on LTLine and LTChar

See the following exceptions :

```python
Traceback (most recent call last):
  File "main.py", line 30, in <module>
    main(sys.argv)
  File "main.py", line 17, in main
    piles = parser.parse()
  File "/Users/bma/git/pdf-to-markdown/pdf2md/parser.py", line 36, in parse
    piles += self._parse_page(page)
  File "/Users/bma/git/pdf-to-markdown/pdf2md/parser.py", line 60, in _parse_page
    pile.parse_layout(page)
  File "/Users/bma/git/pdf-to-markdown/pdf2md/pile.py", line 55, in parse_layout
    assert False, "Unrecognized type: %s" % type(obj)
AssertionError: Unrecognized type: <class 'pdfminer.layout.LTLine'>
```

```python
Traceback (most recent call last):
  File "main.py", line 30, in <module>
    main(sys.argv)
  File "main.py", line 17, in main
    piles = parser.parse()
  File "/Users/bma/git/pdf-to-markdown/pdf2md/parser.py", line 36, in parse
    piles += self._parse_page(page)
  File "/Users/bma/git/pdf-to-markdown/pdf2md/parser.py", line 60, in _parse_page
    pile.parse_layout(page)
  File "/Users/bma/git/pdf-to-markdown/pdf2md/pile.py", line 52, in parse_layout
    assert False, "Unrecognized type: %s" % type(obj)
AssertionError: Unrecognized type: <class 'pdfminer.layout.LTChar'>
```

--- a/pdf2md/pile.py
+++ b/pdf2md/pile.py
@@ -7,6 +7,8 @@ from pdfminer.layout import LTLine
 from pdfminer.layout import LTRect
 from pdfminer.layout import LTImage
 from pdfminer.layout import LTCurve
+from pdfminer.layout import LTChar
+from pdfminer.layout import LTLine
 import binascii
 
 class Pile(object):
@@ -48,6 +50,10 @@ class Pile(object):
 				self.images.append(obj)
 			elif type(obj) == LTCurve:
 				pass
+			elif type(obj) == LTChar:
+				pass
+			elif type(obj) == LTLine:
+				pass					
 			else:
 				assert False, "Unrecognized type: %s" % type(obj)
 

